apps/studio/src/components/tableview/EditorModal.vue

```diff
@@ -194,7 +197,7 @@ export default Vue.extend({
   },
 
   methods: {
-    openModal(content: any, language: LanguageData, eventParams?: any) {
+    openModal(content: any, language: LanguageData, eventParams?: any, isReadOnly?: boolean) {
```

Comment:
Thanks for the PR! Is it possible if we use `eventParams` instead of making a new parameter so we can do `eventParams.isReadOnly`?

Reply:
Awesome!